lib/matplotlib/tests/test_axes.py

```diff
@@ -8290,3 +8290,18 @@ def test_extent_units():
     with pytest.raises(ValueError,
                        match="set_extent did not consume all of the kwargs"):
         im.set_extent([2, 12, date_first, date_last], clip=False)
+
+
+def test_repr_error_message():
+
+    def get_next_color():
+        return 'blue'  # currently unused
```

Comment:
I think it can be marked like this:
```suggestion
        return 'blue'  # pragma: no cover
```

Reply:
Awesome, thanks, I just updated.